modules/libpng/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+    "homepage": "http://www.libpng.org/pub/png/libpng.html",
+    "maintainers": [],
```

Comment:
mind adding yourself as a maintainer for this module? :) this should be very low commitment.

Reply:
Done!